### ISSUE TITLE
The bot needs to adhere to Discord's rate limits

### ISSUE THREAD

**Issue by CleverSource** [TheIcyStar/Roblox-DiscordBotService#1]
Make sure this accepts discord's rate limits, just to advise you. If it doesn't accept the rate limits it'll most likely get the website blocked.

**Comment by TheIcyStar** [comment]
Yes, you're right. This does need some way of managing ratelimits.
A queue system for a big flood of messages would be nice. I guess that will be the next thing to work towards with this bot.
